src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -64,9 +64,12 @@ enums:
 types:
  EntityReference:
     typeof: uriorcurie
-    description: A reference to a mapped entity. This is represented internally as a string, and as a resource in RDF
+    description: |
+      A reference to an entity involved in the mapping.
     base: str
     uri: rdfs:Resource
+    see_also:
+      - https://mapping-commons.github.io/sssom/spec/#tsv
```

Comment:
Just to note that this link _may_ need to change in the future (not sure yet, but after the overhaul the specification for the TSV format will likely be in a dedicated file, rather than in a section inside another file) – but we can change it later if/when needed.